# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/compiler-rt-adjust-paths.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-141.0.7390.54/build/config/clang/BUILD.gn.vanilla	2025-10-01 20:15:11.699618183 +0200
+++ chromium-141.0.7390.54/build/config/clang/BUILD.gn	2025-10-01 20:16:42.621856383 +0200
@@ -171,12 +171,15 @@
       } else if (is_linux || is_chromeos) {
         if (current_cpu == "x64") {
           _dir = "x86_64-unknown-linux-gnu"
+          _suffix = "-x86_64"
         } else if (current_cpu == "x86") {
           _dir = "i386-unknown-linux-gnu"
+          _suffix = "-i386"
         } else if (current_cpu == "arm") {
           _dir = "armv7-unknown-linux-gnueabihf"
         } else if (current_cpu == "arm64") {
           _dir = "aarch64-unknown-linux-gnu"
+          _suffix = "-aarch64"
         } else if (current_cpu == "loong64") {
           _dir = "loongarch64-unknown-linux-gnu"
         } else if (current_cpu == "riscv64") {
@@ -215,7 +218,8 @@
         assert(false)  # Unhandled target platform
       }
 
-      _clang_lib_dir = "$clang_base_path/lib/clang/$clang_version/lib"
+      _clang_lib_dir = "$clang_base_path/lib"
+      _dir = "linux"
       _lib_file = "${_prefix}clang_rt.${_libname}${_suffix}.${_ext}"
       libs = [ "$_clang_lib_dir/$_dir/$_lib_file" ]
     }
